src/__tests__/fireEvent.test.js

```diff
@@ -317,6 +317,36 @@ test('should not fire on box-only pointerEvents View with nested elements', () =
   expect(handlePress).not.toHaveBeenCalled();
 });
 
+test('should fire non-terminal touch events on box-none pointerEvents View', () => {
```

Comment:
The name "non-terminal" was an attempt to communicate the subject even is not "press," with is the "ultimate" or "primary" touch event. Definitely open to feedback on better names.

Reply:
How about `pointer` events? While they are not mentioned directly in React-Native docs (which uses the term 'touch events') it is rather well-defined in the web version of the same concept: https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent